src/differenceWith.js

```diff
@@ -11,7 +11,7 @@ var _curry3 = require('./internal/_curry3');
  * @memberOf R
  * @since v0.1.0
  * @category Relation
- * @sig (a -> a -> Boolean) -> [*] -> [*] -> [*]
+ * @sig ((a, a) -> Boolean) -> [a] -> [a] -> [a]
```

Comment:
as i mentioned below, i think i disagree. Viewed as a "set" relation, I would expect `A - B = A`  where `A` is a set of type `a` and `B` is a set of some other type `B`.